scripts/fdbshow

```diff
@@ -68,8 +72,14 @@ class FdbShow(object):
             fdb = json.loads(fdb_entry .split(":", 2)[-1])
             if not fdb:
                 continue
-
-            ent = self.db.get_all('ASIC_DB', s, blocking=True)
+            if self.db.exists('ASIC_DB',s):
+                try:
+                    ent = self.db.get_all('ASIC_DB', s, blocking=True)
+                except:
```

Comment:
Can we just look for an exception with a log saying some entries are aged-out?. Apparently, this is not a frequent use-case and the user can re-issue the command? Also there is an improvement on 'show mac' to be fetched from STATE_DB

Reply:
@prsunny yes it is not a frequent use-case.
you are suggesting to print as an output when this case occurs that some MAC aged out?
re-issue the command will probably work, but this is expected behaviour?
can you please point to the improvement you are referring to?

Reply:
yes, i was referring to this PR - https://github.com/Azure/sonic-utilities/pull/968

Reply:
@prsunny looks like the PR you mentioned will improve and fix the issue.
Is it going to be merged soon? If it will take a long time to review and merge all PR's needed for this improvement maybe we should merge this fix for now, what do you think?

Reply:
I believe, the other one would be updated and merged soon.

Reply:
OK, thanks